### PR TITLE
FFmpeg non windows system fix

### PR DESCRIPTION
**Pull request by Athkore**
Uses a variable to store ffmpeg executable name, if system is windows, use ffmpeg.exe, else use ffmpeg.
If not Windows, test if ffmpeg can be executed in runtime, and throw error if it can not be found.

### CODE REVIEW

File: src/main/java/com/github/nicholasmoser/audio/FFmpeg.java [hunk 1]
```diff
@@ -17,11 +17,22 @@ public class FFmpeg {
    * @throws IOException If an I/O error occurs.
    */
   public static String prepareSoundEffect(Path input, Path output) throws IOException {
-    if (!Files.isRegularFile(Paths.get("ffmpeg.exe"))) {
-      throw new IOException("ffmpeg.exe cannot be found.");
+    String ffmpeg;
+    if (System.getProperty("os.name").startsWith("Windows")) {
```
Comment (NicholasMoser) [inline src/main/java/com/github/nicholasmoser/audio/FFmpeg.java:21]: Can you replace this with `Platform.isWindows()`? I use that API elsewhere in the codebase and would prefer to keep it consistent.

Comment (Athkore) [reply src/main/java/com/github/nicholasmoser/audio/FFmpeg.java]: Changed it now

**Review by NicholasMoser — APPROVED** [review]

File: src/main/java/com/github/nicholasmoser/audio/FFmpeg.java (hunk 1 above)
Comment (NicholasMoser) [reply src/main/java/com/github/nicholasmoser/audio/FFmpeg.java]: Hmm, just realized that the `jna` dependency is test scope currently:

`testImplementation 'net.java.dev.jna:jna:5.9.0'`

so it's only being used in tests. I'll fix this in a follow-up PR

Comment (NicholasMoser) [reply src/main/java/com/github/nicholasmoser/audio/FFmpeg.java]: https://github.com/NicholasMoser/GNTool/pull/100

**Pull request merged**